README.md

```diff
@@ -2,19 +2,40 @@
 
 # Stay Tune
 
-> React Capstone Project
+> This is my first React Capstone Project. In the project, I build a tv show sites, which gives information about Shows telecast on TV Maze. I choose a dark color theme for this particular project. I have used a shade of black (#1b1b1b). I would like to thank  Nelson Sakwa on Behance for the original design idea.  This app was build using HTML, CSS, JavaScript, React and Redux.
 
 ## Built With
 
 - HTML
 - CSS
 - Javascript
 - React
+- Redux
+
+## Prerequisites
+
+- Good knowledge on javascript objects
+- VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Node package. [Link to download node](https://nodejs.org/en/download/)
+
+## Getting started
+
+to run the project locally, clone the project using the command
+
+- git clone git@github.com:AngomRanjan/stay-tune.git
+- npm install
+
+### Run test
+
+navigate to the project directory, from GitHub,
+to test or run the application double click on the `index.html` file to open in your browser
 
 ## Live Demo
 
 [Live Demo Link](https://stay-tune-angom.netlify.app)
 
+[Project Presentation Video](https://www.loom.com/share/59358e0d86054813a5687fdc44ac7ffd)
+
 ## Authors
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by sole contributor.

Reply:
Thank you, I will change it right away